scss/_patterns_buttons.scss

```diff
@@ -16,14 +15,6 @@ $hover-background-opacity-percentage: $hover-background-opacity-amount * 100%;
   @include vf-button-icon;
 }
 
-@mixin vf-button-white-success-icon {
```

Comment:
I would leave the mixin empty, deprecate it, but keep it for the sake of not breaking the builds that may include it (like cookie policy now).

Something similar to this (but empty):
https://github.com/canonical-web-and-design/vanilla-framework/blob/387102aef5aea9b1aa5dd17f79b4853b4eb650af/scss/_patterns_grid.scss#L3-L9

Reply:
Ah yes - updating